service/src/main/java/org/ehrbase/service/ValidationServiceImp.java

```diff
@@ -131,6 +135,44 @@ public void check(Composition composition) throws Exception {
         check(composition.getArchetypeDetails().getTemplateId().getValue(), composition);
     }
 
+    @Override
+    public void check(EhrStatus ehrStatus) {
+
+        //case of a system generated ehr
+        if (ehrStatus == null)
+            return;
+
+        //first, check the built EhrStatus using the general Archie RM-Validator
```

Comment:
I changed the order of the checks. Now, first, the general RM validation is done and secondly, our more custom rules are applied. Alright?
The other way around I found it hard to differentiate between what I need to keep in mind and check with our `if`s VS. what the archie validator is going to check anyway. Now the broader and general check is applied first and probably catches all usual problems.